python/pyspark/sql/readwriter.py

```diff
@@ -289,8 +289,8 @@ def text(self, paths):
         [Row(value=u'hello'), Row(value=u'this')]
         """
         if isinstance(paths, basestring):
-            path = [paths]
-        return self._df(self._jreader.text(self._spark._sc._jvm.PythonUtils.toSeq(path)))
+            paths = [paths]
+        return self._df(self._jreader.text(self._spark._sc._jvm.PythonUtils.toSeq(paths)))
```

Comment:
This is a super minor but I think it'd be nicer to match up the variable name to `path` if this makes sense. For parquet, it takes non-keyword arguments so it seems `paths` but for others, it seems take a single argument.